dogfooding/lib/widgets/call_duration_title.dart

```diff
@@ -0,0 +1,82 @@
+import 'dart:async';
+
+import 'package:flutter/material.dart';
+import 'package:flutter_dogfooding/utils/assets.dart';
+import 'package:flutter_svg/flutter_svg.dart';
+import 'package:stream_video_flutter/stream_video_flutter.dart';
+
+class CallDurationTitle extends StatefulWidget {
+  const CallDurationTitle({
+    super.key,
+    required this.call,
+  });
+
+  final Call call;
+
+  @override
+  State<CallDurationTitle> createState() => _CallDurationTitleState();
+}
+
+class _CallDurationTitleState extends State<CallDurationTitle> {
+  Duration _duration = Duration.zero;
+  Timer? _timer;
+
+  @override
+  void initState() {
+    super.initState();
+
+    widget.call.get().then((value) {
+      final startedAt = value.foldOrNull(
+              success: (callData) =>
+                  callData.data.metadata.session.startedAt) ??
+          DateTime.now();
+      _duration = DateTime.now().difference(startedAt);
+
+      _timer = Timer.periodic(const Duration(seconds: 1), (timer) {
+        if (!mounted) {
+          timer.cancel();
+          return;
+        }
+
+        setState(() {
+          _duration += const Duration(seconds: 1);
```

Comment:
I think it is more accurate to keep `startedAt` in the state and not the `_duration` and then calculate the `difference`. This way we can provision for any unsynced ticks.

Reply:
yes, agreed. Done